dist/related-items-style.css

```diff
@@ -31,8 +31,12 @@
   padding: 5px;
 }
 
+.hidden {
+  color: white;
+}
+
 .fixedHeight {
-  height: 51.77px;
+  height: 52.82px;
 }
```

Comment:
Why not just set the height of div.card?

Reply:
I think this will allow us to change the font style/height without having to worry about text-overflow. I tested fixed height cards with different text sizes, and if we increase the text size, the text overflows. Might still have to tweak a few things though if we do end up making overall styling changes—so likely minimal gain.